packages/theme/css/characteristics/separable.css

```diff
@@ -4,4 +4,11 @@
   --salt-separable-primary-borderColor: var(--salt-palette-alpha-strong);
   --salt-separable-secondary-borderColor: var(--salt-palette-alpha);
   --salt-separable-tertiary-borderColor: var(--salt-palette-alpha-weak);
+
+  --salt-separable-foreground: var(--salt-palette-neutral-primary-foreground);
+  --salt-separable-foreground-hover: var(--salt-palette-neutral-primary-foreground);
+  --salt-separable-foreground-active: var(--salt-palette-interact-cta-foreground);
+  --salt-separable-background: var(--salt-palette-alpha-none);
+  --salt-separable-background-hover: var(--salt-palette-alpha-weak);
```

Comment:
```suggestion
  --salt-separable-background-hover: var(--salt-palette-alpha-none);
```

Reply:
looking at figma i would have thought it was this one, but not super sure so feel free to ignore me

Reply:
@Fercas123 There's a comment in Figma to figure out which is the right value

Reply:
Just talked with @ivan-calderon, and we concluded that the correct value is `alpha-weak`